Dockerfile-alpine.template

```diff
@@ -81,19 +81,21 @@ RUN set -ex; \
     \
     export GNUPGHOME="$(mktemp -d)"; \
     export GPGKEY="3D06A59ECE730EB71B511C17CE752F178259BD92"; \
-    curl -fsSL -o phpMyAdmin.tar.xz $URL; \
-    curl -fsSL -o phpMyAdmin.tar.xz.asc $URL.asc; \
+    curl -fsSL -o phpMyAdmin-$VERSION-all-languages.tar.xz $URL; \
+    curl -fsSL -o phpMyAdmin-$VERSION-all-languages.tar.xz.asc $URL.asc; \
+    curl -fsSL -o phpMyAdmin-$VERSION-all-languages.tar.xz.sha256 $URL.sha256; \
```

Comment:
Downloading the checksum during _build_ doesn't provide any value (as you've pointed out).  This should be downloaded during update and embedded into the templated file.

Reply:
That actually makes more sense.